src/QsCompiler/Compiler/Templates/QirDriverCpp.tt

```diff
@@ -1,16 +1,268 @@
-﻿<#@ template language="C#" linePragmas="false" #>
+﻿<#@ template language="C#" #>
```

Comment:
```suggestion
<#@ template language="C#" linePragmas="false" #>
```

Reply:
This is needed such that the the generated C# code does not have `#line` pragmas.

Reply:
I see, added back in